repo-index.yml

```diff
@@ -951,6 +951,25 @@ plugins:
   name: network-policy
   updated: 2017-11-03T18:54:18Z
   version: 1.6.1
+- authors:
+  - name: CF Loggregator Team
+  binaries:
+  - checksum: 92be3d4e49d174748652edbf68ef1d3c0c4deb74
+    platform: osx
+    url: https://github.com/cloudfoundry/noisy-neighbor-nozzle/releases/download/v1.5/noisy-neighbor-cli-plugin-darwin
+  - checksum: 6609346819056b3e25a4bbd9f1132e08e529591d
+    platform: win64
+    url: https://github.com/cloudfoundry/noisy-neighbor-nozzle/releases/download/v1.5/noisy-neighbor-cli-plugin-windows
+  - checksum: d13f2ce90a28f44f511fcd8172955f7ed221dd7d
+    platform: linux64
+    url: https://github.com/cloudfoundry/noisy-neighbor-nozzle/releases/download/v1.5/noisy-neighbor-cli-plugin-linux
+  company: Pivotal
+  created: 2018-04-20T00:00:00Z
+  description: A nozzle and CLI tool for identify high volume log producing applications
+  homepage: https://github.com/cloudfoundry/noisy-neighbor-nozzle
+  name: noisy-neighbor-nozzle
```

Comment:
When I install the plugin, it's called `Log Noise CLI Plugin` but the name you're providing is `noisy-neighbor-nozzle`. We're now asking for plugin developers to be consistent with the naming as we've had confusion with this in the past:
```
cf install-plugin -f https://github.com/cloudfoundry/noisy-neighbor-nozzle/releases/download/v1.5/noisy-neighbor-cli-plugin-linux
Attention: Plugins are binaries written by potentially untrusted authors.
Install and use plugins at your own risk.
Starting download of plugin binary from URL...
 8.04 MiB / 8.04 MiB [=======================================================================================================] 100.00% 8s
Installing plugin Log Noise CLI Plugin...
OK
Plugin Log Noise CLI Plugin 1.5.0 successfully installed.
```

Reply:
@XenoPhex if it's officially called `Log Noise CLI Plugin`, is the name sufficiently synced if named `log-noise`?